core/src/Kokkos_CopyViews.hpp

```diff
@@ -56,11 +56,27 @@ namespace Impl {
 
 template <class ViewType, class Layout, class ExecSpace, typename iType>
 struct ViewFill<ViewType, Layout, ExecSpace, 0, iType> {
+  ViewType a;
```

Comment:
hmm ... so we hope on the compiler figuring out that it is unused in some cases. I guess it is fine